package/com.unity.formats.usd/Editor/Scripts/Behaviors/UsdMenu.cs

```diff
@@ -19,90 +19,118 @@
-
-    public static Scene InitForSave(string defaultName, string fileExtension = "usd,usda,usdc") {
-      var filePath = EditorUtility.SaveFilePanel("Export USD File", "", defaultName, fileExtension);
-
-      if (filePath == null || filePath.Length == 0) {
-        return null;
-      }
-
-      var fileDir = Path.GetDirectoryName(filePath);
-
-      if (!Directory.Exists(fileDir)) {
-        var di = Directory.CreateDirectory(fileDir);
-        if (!di.Exists) {
-          Debug.LogError("Failed to create directory: " + fileDir);
-          return null;
+namespace Unity.Formats.USD
+{
+    public class UsdMenu : MonoBehaviour
+    {
+        const string PayloadPolicyPrefName = "USD.LoadAllPayloads";
+        const string MaterialPolicyPrefName = "USD.LoadPreviewSurface";
+
+        const string UsdPrefPayloadMenuName = "USD/Preferences/Load all payloads";
+        const string UsdPrefMaterialMenuName = "USD/Preferences/Load preview surface";
+
+        public static Scene InitForSave(string defaultName, string fileExtension = "usd,usda,usdc")
+        {
+            var filePath = EditorUtility.SaveFilePanel("Export USD File", "", defaultName, fileExtension);
+
+            if (filePath == null || filePath.Length == 0)
+            {
+                return null;
+            }
+
+            var fileDir = Path.GetDirectoryName(filePath);
+
+            if (!Directory.Exists(fileDir))
+            {
+                var di = Directory.CreateDirectory(fileDir);
+                if (!di.Exists)
+                {
+                    Debug.LogError("Failed to create directory: " + fileDir);
+                    return null;
+                }
+            }
+
+            InitUsd.Initialize();
+            var scene = Scene.Create(filePath);
+            scene.Time = 0;
+            scene.StartTime = 0;
+            scene.EndTime = 0;
+            return scene;
         }
-      }
-
-      InitUsd.Initialize();
-      var scene = Scene.Create(filePath);
-      scene.Time = 0;
-      scene.StartTime = 0;
-      scene.EndTime = 0;
-      return scene;
-    }
 
-    public static Scene InitForOpen() {
-      string path = EditorUtility.OpenFilePanel("Import USD File", "", "usd,usda,usdc,abc");
-      if (path == null || path.Length == 0) {
-        return null;
-      }
-      InitUsd.Initialize();
-      var stage = pxr.UsdStage.Open(path, pxr.UsdStage.InitialLoadSet.LoadNone);
-      return Scene.Open(stage);
-    }
-
-    [MenuItem("USD/Export Selected with Children", true)]
-    static bool EnableMenuExportSelectedWithChildren() {
-      return Selection.gameObjects.Length > 0;
-    }
-    [MenuItem("USD/Export Selected with Children", priority = 50)]
-    static void MenuExportSelectedWithChildren() {
-      ExportSelected(BasisTransformation.SlowAndSafe);
-    }
+        public static Scene InitForOpen()
+        {
+            string path = EditorUtility.OpenFilePanel("Import USD File", "", "usd,usda,usdc,abc");
```

Comment:
I suggest strongly to not change massively the indentation during a feature PR: close to impossible to see the actual changes.

Reply:
+1 -- if you agree, I'm actually going to hold off on reviewing until after that

Reply:
Isn't it what the commit selection menu is for? Or at least it's how I used it before. 
I can get rid of the reformat in this PR and just do a pass on the whole package instead. Would you prefer that?

Reply:
Unless the reformatting is really minor, I usually break it into its own PR

But the second issue is that this is also reformating this one bit of code with a different standard from the rest of the repository.

How about this: I think this PR is fine as-is (it looks like the reformatting is in its own commit already), but can you do a follow up PR that reformats the rest of the repo to Unity's formatting conventions?

Reply:
Removed the formatting commit and pushed a new formatting PR #229